src/main/java/org/spongepowered/common/mixin/core/tileentity/MixinTileEntityMobSpawner.java

```diff
@@ -58,6 +59,6 @@ public void spawnEntityBatchImmediately(boolean force) {
     @Override
     public void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators) {
         super.supplyVanillaManipulators(manipulators);
-        // TODO manipulators.add(getMobSpawnerData());
+        manipulators.add(getMobSpawnerData());
```

Comment:
Provide a default implementation of `getMobSpawnerData()` so that there's minimal overhead of map lookups and abstraction layers ;).